src/behavior-considered-undefined.md

```diff
@@ -1,9 +1,14 @@
 ## Behavior considered undefined
 
-Rust code, including within `unsafe` blocks and `unsafe` functions is incorrect
-if it exhibits any of the behaviors in the following list. It is the
-programmer's responsibility when writing `unsafe` code that it is not possible
-to let `safe` code exhibit these behaviors.
+Rust code is incorrect if it exhibits any of the behaviors in the following
+list. This includes code within `unsafe` blocks and `unsafe` functions!
+`unsafe` is not an excuse to cause undefined behavior.
+
+It is the programmer's responsibility when writing `unsafe` code that it is not
+possible to let `safe` code exhibit these behaviors. `unsafe` code that
```

Comment:
```suggestion
possible for safe code exhibit these behaviors for any input, return value, or program state. `unsafe` code that
```
- there's no `safe` in code

Reply:
True, I just stuck to existing terminology there.

The enumeration "input, return value or program state" is very odd though. "For any safe code interacting with it" or so would be better IMO?

Reply:
The main thing I wanted to get at is that e.g.
```rust
fn foo(x: u8) {
    if x < 5 { unsafe { trigger_ub(); } }
}
```
is not sound because it can cause UB for some inputs.

That is, I wanted to point at the "universal quantifier" here.

> "For any safe code interacting with it" or so would be better IMO?

Yeah I suppose that is mostly the same thing. I think that's good.